server/__init__.py

```diff
@@ -21,5 +23,7 @@ def load(info):
 
     genRESTEndPointsForSlicerCLIsInDocker(
         info, 'HistomicsTK',
-        'dsarchive/histomicstk:v0.1.2'
+        'jbeezley/histomicstk'
```

Comment:
I see that you already have owner rights for the dsarchive organization on Dockerhub.

Currently, Dockerhub automatically: 
* builds the dsarchive/histomicstk:latest image whenever a commit is merged into master
* builds the dsarchive/histomicstk:<release-tag> whenever a release is made for the HistomicsTK repo.

Once you merge these changes and are sure the end-to-end pipeline for NucleiSegmentation CLI (Select input image, parameters, set output file location and names through front-end, Submit job, Auto ingest the generated annotation file into girder, render the results) then create a new release and update the release tag here which will trigger the autobuild on DockerHub.